index.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+  
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" href="style.css">
+      <title>Project</title>
```

Comment:
Please give meaningful names to your Projects. Meaningful names can clearly distinguish one project from another.

Reply:
Your comments are polite and you use why +1